linux_proc_extras/tests/conftest.py

```diff
@@ -15,4 +15,4 @@ def dd_environment():
 
 @pytest.fixture
 def check():
-    return MoreUnixCheck(common.CHECK_NAME, {}, {})
+    return MoreUnixCheck(common.CHECK_NAME, {}, {}, [common.INSTANCE])
```

Comment:
Passing agentConfig is deprecated
```suggestion
    return MoreUnixCheck(common.CHECK_NAME, {}, [common.INSTANCE])
```